tests/e2e/e2e_util_test.go

```diff
@@ -188,14 +220,25 @@ func (s *IntegrationTestSuite) depositProposal(c *chain.Chain) {
 
 	s.Require().Truef(
 		strings.Contains(outBuf.String(), "code: 0"),
-		"tx returned non code 0",
+		"tx returned non code 0; stdout: %s, stderr: %s", outBuf.String(), errBuf.String(),
+	)
+
+	currentHeight := s.chainHeight(s.valResources[c.ChainMeta.Id][0].Container.ID)
+
+	s.Require().Eventually(
```

Comment:
What is the purpose of this?

Reply:
Without this, you run into account sequence mismatches. The only way around this would be to round robin multiple wallets on the hermes docker container for chain A and chain B instead of only using 1 for each.

Reply:
So I tested out the round robin wallet idea. While it ended up working, it felt overly convoluted. I realized the best solution was an extremely obvious one; spam txs with a single account until you get a successful send code. Will post what I have soon